torch/_inductor/kernel/mm.py

```diff
@@ -133,14 +133,31 @@ def tuned_mm(mat1, mat2, *, layout=None):
             )
 
     if m * n != 0 and use_cutlass_template(layout):
-        cutlass_template = CUTLASSGemmTemplate([mat1, mat2], layout, alpha=1, beta=0)
+        cutlass_template = CUTLASSGemmTemplate(
```

Comment:
I think we probably only need to keep templates which support EVT.
Reasons are:
1) According to Vijay, for most cases we only need to keep TMA mainloop configs which support EVT (https://fburl.com/gdoc/575xeqlt).
2) For templates which do not support EVT, it's likely that CUBLAS would provide similar perf (since epilogue fusion cannot be done).

Reply:
If I make this change, I think many preexisting tests in test_max_autotune.py will fail due to lacking op candidates. The EVT capable ops support only fp16 output at the moment.

Reply:
as discussed, fp32 accumulation is supported, but there are no evt capable kernels being generated / enumerated by the cutlass_library functions. But that's likely not a fundamental issue.